docs/data/base/getting-started/overview/overview.md

```diff
@@ -2,19 +2,17 @@
 title: Overview
 ---
 
-# Base UI - Overview
+# Overview
```

Comment:
I think we added Base UI as a cheap workaround waiting to fix the header:

<img width="581" alt="SCR-20240421-ospa" src="https://github.com/mui/base-ui/assets/3165635/ec8ed10a-3cf6-4bb9-8ade-15f67600b47c">

to make it clear it's not some other library. If we make this change, we do no longer need a custom SEO `<title>`, we can remove it.

```
---
title: Overview
---
```